### PR TITLE
Dev -> master for v1.2.0

### PR DESCRIPTION
**Pull request by pinin4fjords**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/differentialabundance/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/differentialabundance _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: assets/differentialabundance_report.Rmd [hunk 1]
```diff
@@ -139,59 +144,6 @@ library(DT)
 datatable(NULL)
 ```
 
-```{r, echo=FALSE}
```
Comment (pinin4fjords) [inline assets/differentialabundance_report.Rmd:142]: Reviewers: this is just because the function was moved up into shinyngs

Comment (maxulysse) [reply assets/differentialabundance_report.Rmd:142]: I appreciate such comments

**Review by maxulysse — APPROVED** [review]
<3

File: README.md [hunk 2]
```diff
@@ -73,6 +74,26 @@ Affymetrix microarray:
...
+
+#### Shiny-based data mining app
+
+A second optional output is produced by leveraging [shinyngs](https://github.com/pinin4fjords/shinyngs) to build an interactive Shiny application. This allows more interaction with the data, setting of thresholds etc.
+
+![screenshot of the ShinyNGS contrast table](docs/images/shinyngs_contrast_table.png "ShinyNGS contrast table")
+
+![screenshot of the ShinyNGS gene plot](docs/images/shinyngs_gene_plot.png "ShinyNGS gene plot")
+
+By default the application is provided as an R script and associated serialised data structure, which you can use to quickly start an the application locally. With proper configuration the app can also be deployed to [shinyapps.io](https://www.shinyapps.io/) - though this requires you to have an account on that service (free tier available).
```
Comment (pinin4fjords) [inline README.md:95]:
```suggestion
By default the application is provided as an R script and associated serialised data structure, which you can use to quickly start the application locally. With proper configuration the app can also be deployed to [shinyapps.io](https://www.shinyapps.io/) - though this requires you to have an account on that service (free tier available).
```

File: nextflow_schema.json [hunk 3]
```diff
@@ -778,12 +783,51 @@
...
+                "shinyngs_shinyapps_account": {
+                    "type": "string",
+                    "default": "None",
+                    "description": "Your shinyapps.io account name",
+                    "fa_icon": "fas fa-user"
+                },
+                "shinyngs_shinyapps_app_name": {
+                    "type": "string",
+                    "default": "None",
+                    "description": "The name of the app you crated in your shinyapps.io account",
```
Comment (pinin4fjords) [inline nextflow_schema.json:819]:
```suggestion
                    "description": "The name of the app you created in your shinyapps.io account",
```

**Review by WackerO — APPROVED** [review]

**Comment by pinin4fjords** [comment]
Thanks for review @maxulysse @WackerO

**Pull request merged**